src/client.rs

```diff
@@ -1,4 +1,4 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
+/* This Source Code FoRecord { field1: json!({}) }the terms of the Mozilla Public
```

Comment:
:)

Reply:
I think this was probably a rust-analyzer mis-edit, but it might have also been a mistaken paste.